package.json

```diff
@@ -56,20 +56,26 @@
     "source-map-support": "~0.5.12",
     "ts-jest": "^24.0.2",
     "ts-node": "^8.2.0",
-    "tslint": "~5.17.0"
+    "vega-lite-dev-config": "^0.1.7"
```

Comment:
It's your fork and therefor you can do whatever you want with it of course but I have to comment on it anyway:

As someone who is not a Vega developer, not even a Vega user, I had the impression that ts-json-schema-generator is a standalone project, a generic tool for converting generic TypeScript data structures into a JSON schema. This is not a fork with vega-specific changes only used for vega-lite, instead the tool is published under its own name, not related to Vega, on NPM, so others can easily use it for their non-Vega projects. And there is even an open PR for merging this project back into the upstream project from mrix who is (as far as I know) not related to Vega at all. 

With all this in mind it feels wrong to make this project dependent on some configuration from vega-lite. Even when its only a dev dependency it feels wrong. Some linter configurations done for vega-lite must not necessarily make sense for the schema generator and vice versa. So wouldn't it be better to simply add the eslintrc file here and use eslint directly without beemo and without vega-lite-dev-config?

I really like the current structure of the project and the simplicity of the build which simply directly calls the corresponding standard tools (tsc, tslint, jest) and that's it. No gulp/grunt/burp/fart magic. Its easy to understand and easy to use. Just take a look in the package.json and you know what's going on and which file you have to change to change a linter rule for example. So I would prefer keeping it that way and not adding auto-generated configurations coming from an external dependency.

But as I said, it's your fork, so it's up to you.

Reply:
I also think that the schema generator should be a project that is somewhat independent from Vega. The goal os using the VL dev config was to just make my life easier when we have a single shared well-configured setup. You will see that beemo actually copies the eslint/prettier files so it should be quite easy to remove beemo and vl dev config. I am happy to do that once we have figured out a good config in a few months. @kanitw is still working on figuring out the right config.

Reply:
Done